changelog.md

```diff
@@ -1,4 +1,56 @@
- <a name="7.1.0"></a>
+<a name="7.2.0"></a>
+# 7.2.0 (2021-02-24)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v7.1.0...v7.2.0)
+
+We expect this release to ship in the DevTools of [Chrome 90](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻!
+
+- @1d0n7kn0w
+- Benajmin Franz @BennyAlex
+
+## Core
+
+* emulation: bump chrome versions ([#12104](https://github.com/GoogleChrome/lighthouse/pull/12104))
+* unsized-images: ignore non-network SVGs ([#12120](https://github.com/GoogleChrome/lighthouse/pull/12120))
+* preload: do not warn for cache loads ([#12122](https://github.com/GoogleChrome/lighthouse/pull/12122))
+* url: redirects-http => n/a for localhost. refactor url special cases ([#11766](https://github.com/GoogleChrome/lighthouse/pull/11766))
```

Comment:
don't even know what to do with this commit message :)

Reply:
i just changed it